doc/release-notes/6.3-release-notes.md

```diff
@@ -0,0 +1,475 @@
+# Dataverse 6.3
+
+Please note: To read these instructions in full, please go to https://github.com/IQSS/dataverse/releases/tag/v6.3 rather than the list of releases, which will cut them off.
+
+This release brings new features, enhancements, and bug fixes to Dataverse. Thank you to all of the community members who contributed code, suggestions, bug reports, and other assistance across the project.
+
+# Table of Contents
+- [Release Highlights](#release-highlights)
+- [Features](#features)
+- [Bug Fixes](#bug-fixes)
+- [API](#api)
+- [Settings](#settings)
+- [Complete List of Changes](#complete-list-of-changes)
+- [Getting Help](#getting-help)
+- [Upgrade instructions](#upgrade-instructions)
+
+## Release Highlights
+
+### Solr Search and Indexing Improvements
+
+Multiple improvements have been made to the way Solr indexing and searching is done. Response times should be significantly improved.
```

Comment:
"dramatically improved" ! - I don't know what the final results were but things are multiple times faster now - seems like significantly improved isn't enough.

Reply:
I actually like significantly improved. I'd rather underpromise and over deliver. We can always tweak the release note post-release.